style.css

```diff
@@ -0,0 +1,98 @@
+* {
+  margin: 0;
+  padding: 0;
+  -webkit-box-sizing: border-box;
+  -moz-box-sizing: border-box;
+  box-sizing: border-box;
+}
+
+body {
+  top: 20px;
+  font-family: Poppins, sans-serif;
+  background-color: #1c1a19;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: #3c3a39;
+  padding: 20px;
+}
+
+a {
+  text-decoration: none;
+  color: white;
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 18px;
+  line-height: 24px;
+}
+
+.menu {
+  color: white;
+  cursor: pointer;
+}
+
+.name {
+  cursor: pointer;
+}
+
+.headline {
+  background-image: url(img/headline-illustration.png);
+  background-repeat: no-repeat;
+  background-position: 100% 90%;
```

Comment:
I suggest you compare the position of this background image, yours: [Live demo](https://rawcdn.githack.com/2bleO/my-portfolio/bf47465b27b3c710e2ae1872d483766b41cd4b66/index.html) with [Figma Design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A27) and make the necessary adjustments

Reply:
Hi @Davidosky007!

May you please be a bit more specific because I can't spot any issue with the background image position.

Reply:
# This is how your project looks 
![Screenshot (13)](https://user-images.githubusercontent.com/52497246/120777808-976a9480-c51d-11eb-890d-c29184c52009.png) 
# This is the figma design
![Screenshot (14)](https://user-images.githubusercontent.com/52497246/120778249-0647ed80-c51e-11eb-9561-2e1eefbb80b3.png)

# compare the two fix, the differences and submit for review. Thanks 🙏

Reply:
That isn't how it looks!
This is how it looks:
![image](https://user-images.githubusercontent.com/64618543/120780899-8bb7a780-c4f6-11eb-9a2f-2067a7f1daaf.png)

Reply:
When you open it, it opens in desktop mode, right after opening the dev tool it looks the way you say it looks, but that it's just a browser glitch. You have to refresh the page after opening the dev tool.

Reply:
@2bleO you are good to go 👍 the design has a width of 375px while my browser was set to 320px that's why it looked that way sorry for the misunderstanding 🙏 you can submit 💯